docs/05-technical-reference/00-custom-resources/telemetry-03-tracepipeline.md

```diff
@@ -43,44 +43,56 @@ For details, see the [TracePipeline specification file](https://github.com/kyma-
 <!-- The content between "TABLE-START" and "TABLE-END" will be replaced -->
 
 <!-- TABLE-START -->
+### TracePipeline.v1alpha1.telemetry.kyma-project.io
+**Spec:**
+
+<!-- TracePipeline v1alpha1 telemetry.kyma-project.io -->
+| Parameter         | Type | Description                                   |
+| ------------------| ---- | --------------------------------------------- |
+| **spec.output** | object | Defines a destination for shipping trace data. Only one can be defined per pipeline. |
```

Comment:
Can you remove the "spec." prefix, it is redundant and will reduce the width of the column

Reply:
sure